MapboxCoreNavigation/RouteController.swift

```diff
@@ -123,6 +123,12 @@ open class RouteController: NSObject {
      If true, every 2 minutes the `RouteController` will check for a faster route for the user.
      */
     public var checkForFasterRouteInBackground = false
+    
+    
+    /**
+     If true, the spoken voice instructions for the route will be displayed on the map at their locations.
+     */
+    public var showDebugSpokenInstructionsOnMap = false
```

Comment:
The map SDK has an [`MGLDebugMaskOptions` enumeration](https://www.mapbox.com/ios-sdk/api/3.6.4/Enums/MGLMapDebugMaskOptions.html) for things like this. An enumeration would be preferable to allowing a series of debug-only flags to pollute the namespace.